### PR TITLE
Add alternative instructions for ENTRYPOINT in serverless-init

### PR DESCRIPTION
**Pull request by IvanTopolcic**
### What does this PR do?

Adds alternative instructions for using CMD instead of ENTRYPOINT for instrumenting serverless-init.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by tianchu — APPROVED** [review]
Giving my approval! But I wonder if we could fit the alternative solution in the main section instead of a new section at the bottom of the page.

File: content/en/serverless/azure_container_apps/_index.md [hunk 1]
```diff
@@ -459,6 +459,31 @@ Metrics are calculated based on 100% of the application’s traffic, and remain
 | `DD_SOURCE`       | See [Unified Service Tagging][5].                                       |
 | `DD_TAGS`         | See [Unified Service Tagging][5].                                       |
 
+## Alternative configurations
+
+### Alternatives to entrypoint
+
+If you have already defined an entrypoint inside of your Dockerfile, you can instead modify the CMD argument.
```
Comment (estherk15) [inline content/en/serverless/azure_container_apps/_index.md]:
```suggestion
If you have already defined an entrypoint inside of your Dockerfile, you can modify the CMD argument instead.
```

File: content/en/serverless/azure_container_apps/_index.md [hunk 2]
```diff
@@ -459,6 +459,31 @@ Metrics are calculated based on 100% of the application’s traffic, and remain
...
+
+**Note**: As long as your command to run is passed as an argument to datadog-init, you will receive full instrumentation.
```
Comment (estherk15) [inline content/en/serverless/azure_container_apps/_index.md]:
```suggestion
**Note**: To receive full instrumentation, pass the command to run as an argument to `datadog-init`.
```

File: content/en/serverless/azure_container_apps/_index.md [hunk 3]
```diff
@@ -459,6 +459,31 @@ Metrics are calculated based on 100% of the application’s traffic, and remain
...
+
+The below code snippet illustates how to modify your Dockerfile to instrument Container Apps without modifying your entrypoint.
```
Comment (estherk15) [inline content/en/serverless/azure_container_apps/_index.md]:
```suggestion
The following code snippets show the original Dockerfile and then the modified Dockerfile to instrument Container Apps without modifying your entrypoint.
```

File: content/en/serverless/google_cloud_run/_index.md [hunk 4]
```diff
@@ -478,6 +478,30 @@ Once the deployment is completed, your metrics and traces are sent to Datadog. I
 | `DD_SOURCE`       | See [Unified Service Tagging][6].                                  |
 | `DD_TAGS`         | See [Unified Service Tagging][6].                                  |
 
+## Alternative configurations
+
+### Alternatives to entrypoint
+
+If you have already defined an entrypoint inside of your Dockerfile, you can instead modify the CMD argument.
```
Comment (estherk15) [inline content/en/serverless/google_cloud_run/_index.md]:
```suggestion
If you have already defined an entrypoint inside of your Dockerfile, you can modify the CMD argument instead.
```

File: content/en/serverless/google_cloud_run/_index.md [hunk 5]
```diff
@@ -478,6 +478,30 @@ Once the deployment is completed, your metrics and traces are sent to Datadog. I
...
+
+**Note**: As long as your command to run is passed as an argument to datadog-init, you will receive full instrumentation.
```
Comment (estherk15) [inline content/en/serverless/google_cloud_run/_index.md]:
```suggestion
**Note**: To receive full instrumentation, pass the command to run as an argument to `datadog-init`.
```

File: content/en/serverless/google_cloud_run/_index.md [hunk 6]
```diff
@@ -478,6 +478,30 @@ Once the deployment is completed, your metrics and traces are sent to Datadog. I
...
+
+The below code snippet illustates how to modify your Dockerfile to instrument Cloud Run without modifying your entrypoint.
```
Comment (estherk15) [inline content/en/serverless/google_cloud_run/_index.md]:
```suggestion
The following code snippets show the original Dockerfile and then the modified Dockerfile to instrument Cloud Run without modifying your entrypoint.
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**